src/js/components/Button/stories/IconColor.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Add } from 'grommet-icons';
+
+import { Box, Button, Grommet } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from '../../../utils';
+
+const customButtonIconColor = deepMerge(grommet, {
+  button: {
+    color: 'accent-2',
+  },
+});
+
+const CustomIconLabelColor = () => (
```

Comment:
I think storybook is already too busy on the Button component, and we can use the existing storybook https://storybook.grommet.io/?path=/story/button--icon-label to add a couple of relevant examples there instead of this story.

Reply:
@ShimiSun If I use this storybook I would have a custom theme so it would change some of the icon colors that already exist on that story

Reply:
That would be fine. go for it.